src/compas_fab/robots/robot.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, model, artist=None, semantics=None, client=None):
         self.artist = artist
         self.semantics = semantics
         self.client = client
-        self.pybullet_uid = None
+        self.attributes = {}
```

Comment:
Moved `pybullet_uid` to attributes, so that `Robot` doesn't need to know about it.